src/passkit.cs

```diff
@@ -1316,4 +1372,379 @@ interface PKSecureElementPass {
 		[NullAllowed, Export ("pairedTerminalIdentifier")]
 		string PairedTerminalIdentifier { get; }
 	}
+
+	[NoWatch, NoTV]
+	[iOS (14,0)]
+	[Native]
+	public enum PKAddShareablePassConfigurationPrimaryAction : ulong {
+		Add,
+		Share,
+	}
+
+	[Watch (7,0), iOS (14,0)]
+	[NoTV]
+	[Native]
+	public enum PKBarcodeEventConfigurationDataType : long {
+		Unknown,
+		SigningKeyMaterial,
+		SigningCertificate,
+	}
+
+	[NoWatch, NoTV, NoMac]
+	[iOS (14,0)]
+	[Native]
+	public enum PKIssuerProvisioningExtensionAuthorizationResult : long {
+		Canceled,
+		Authorized,
+	}
+
+	[NoWatch, NoTV]
+	[iOS (13,4)]
+	[Mac (11,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PKAddSecureElementPassConfiguration {
+
+		[NullAllowed, Export ("issuerIdentifier")]
+		string IssuerIdentifier { get; set; }
+
+		[NullAllowed, Export ("localizedDescription")]
+		string LocalizedDescription { get; set; }
+	}
+
+	[NoWatch, NoTV]
+	[Mac (11,0)]
+	[iOS (13,4)]
+	[BaseType (typeof (PKAddSecureElementPassConfiguration))]
+	// note: `init` is present in headers
+	interface PKAddCarKeyPassConfiguration {
+
+		[Export ("password")]
+		string Password { get; set; }
+	}
+
+	interface IPKAddSecureElementPassViewControllerDelegate {}
+
+	[NoWatch, NoTV, NoMac]
+	[iOS (13,4)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject))]
+	interface PKAddSecureElementPassViewControllerDelegate {
+
+		[Deprecated (PlatformName.iOS, 14,0, message: "Use 'DidFinishAddingSecureElementPasses' instead.")]
+		[Abstract]
+		[Export ("addSecureElementPassViewController:didFinishAddingSecureElementPass:error:")]
+		void DidFinishAddingSecureElementPass (PKAddSecureElementPassViewController controller, [NullAllowed] PKSecureElementPass pass, [NullAllowed] NSError error);
+	
+#if XAMCORE_4_0
```

Comment:
This is a new protocol, so you don't need the `#if XAMCORE_4_0` here.

Reply:
no, that's `PKAddSecureElementPassViewControllerDelegate` and was added in iOS 13.4

Reply:
iOS added it in 13.4, but as far as I can see we're adding the bindings for it in this PR, so we don't need to be compatible with existing bindings. Or am I missing something?

Reply:
> Or am I missing something?

Unlikely :) That was added in a different pass and the bindings have been done over a too-long period. I'll double check the versions since I don't recall missing stuff from 13.4.

Reply:
The availability was added in beta 1 (it did not have any before) so it was present in Xcode 11.5

It was added in Xcode 11.4 beta 3 (it's in the diff) but https://github.com/xamarin/xamarin-macios/pull/8041/files does not have it and xtro did not complain (looks like the specific header file was not (at that time) part of the umbrella header). c.c. @dalexsoto